templates/ui-frameworks/svelte/collection/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{pascal_case collection_name}}.svelte.hbs

```diff
@@ -1,71 +1,78 @@
 <script lang="ts">
-import { onMount, getContext } from 'svelte';
-import type {
-  EntryHash,
-  Record,
-  AgentPubKey,
-  ActionHash,
-  Link,
-  AppClient,
-  NewEntryAction,
-  HolochainError,
-} from '@holochain/client';
-import { type ClientContext, clientContext } from '../../contexts';
+import { onMount } from 'svelte';
+import type { EntryHash, Record, AgentPubKey, ActionHash, Link, NewEntryAction, HolochainError } from '@holochain/client';
+import { SignalType } from '@holochain/client';
```

Comment:
`SignalType` was removed from holochain-client-js due to changes in enum serialization: see https://github.com/holochain/holochain-client-js/pull/322/files#

Reply:
Removed, this was a left over change while fixing merge conflicts